hoot-core/src/main/cpp/hoot/core/io/OsmPbfReader.cpp

```diff
@@ -1126,6 +1126,7 @@ void OsmPbfReader::parse(istream* strm, const OsmMapPtr& map)
     // if we don't recognize the type
     else
     {
+      LOG_WARN("Skipping PBF blob type " << _d->blobHeader.type());
```

Comment:
Still reading malformed `.osm.pbf` like we were before but dropping a warning when it is read.